### ISSUE TITLE
ValueError: DataFrame index must be unique for orient='index'.

### ISSUE THREAD

**Issue by baha2180** [architecture-building-systems/CityEnergyAnalyst#2988]
**Describe the bug**
I am getting error - DataFrame index must be unique - python ver 3.9.5
I am totally new in programming as well as in python programming 
Guys It would be great if you guys can help me to resolve below issue
**To Reproduce**

import csv, json
import pandas as pd
#for kiosk
csvpath_kiosk = "consolidated_kiosk.csv"
jsonpath_kiosk = "sales_analytics_report_combine_kiosk.json"

df = pd.read_csv(csvpath_kiosk)

j = df.groupby(['store_id'])[['Date and Time','totalamount', 'totalorder', 'totalqty']]\
.apply(lambda x: x.set_index('Date and Time').to_dict(orient='index')).to_dict()
json_kiosk = json.dumps(j)
f = open(jsonpath_kiosk,'w')
f.write(json_kiosk)
f.close()

**Expected behaviour**
It should have create json file - 
**Screenshots**
![image](https://user-images.githubusercontent.com/84909592/119816717-a98b7800-bf0a-11eb-9ada-4206488a4e56.png)
**Hardware (please complete the following information):**
 - OS and version: [e.g. iOS 10.7 or Windows 10]
 - Version of CEA you are using [e.g. v. 2.31.1]

**Comment by shanshanhsieh** [comment]
Hi @baha2180,

which script are you referring to?

**Comment by baha2180** [comment]
@shanshanhsieh - Sorry but I didn't understand your question related to script? could you please elaborate in layman language?

**Comment by shanshanhsieh** [comment]
hi @baha2180,

I am sorry but we cannot provide support to scripts beyond CEA source code.
